src/ios/Fingerprint.swift

```diff
@@ -16,7 +16,7 @@ import LocalAuthentication
             biometryType = "none";
         }
 
-        var pluginResult = CDVPluginResult(status: CDVCommandStatus_ERROR, messageAs: "Not available");
+        var pluginResult = CDVPluginResult(status: CDVCommandStatus_ERROR, messageAs: "Error: \(String(describing: error?.localizedDescription))");
```

Comment:
I would leave out the `Error: ` because this is already the error callback.

```suggestion
        var pluginResult = CDVPluginResult(status: CDVCommandStatus_ERROR, messageAs: "\(String(describing: error?.localizedDescription))");
```